lib/send_sms.rb

```diff
@@ -0,0 +1,21 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'time'
+
+class SMS
+  def client
+    account_sid = ENV['TWILIO_ACCOUNT_SID']
+    auth_token = ENV['TWILIO_AUTH_TOKEN']
```

Comment:
Great use of environment variables - do you have a resource that you can recommend for using them?